Dockerfile

```diff
@@ -18,4 +19,5 @@ LABEL io.k8s.display-name="OpenShift elasticsearch-operator" \
       io.k8s.description="This is the component that manages an Elasticsearch cluster on a kubernetes based platform" \
       io.openshift.tags="openshift,logging,elasticsearch" \
       io.openshift.release.operator=true \
```

Comment:
you still need to remove this...

Reply:
https://github.com/openshift/elasticsearch-operator/pull/100

Reply:
why is that a separate PR though? it seems like we're going to have an issue if we have both this label and `com.redhat.delivery.appregistry=true` at the same time...

Reply:
well the issue will be that this won't pass tess until that line is removed.  :)